### DISCUSSION TITLE
The Vuetify 3 Beta website can't show guide

### DISCUSSION THREAD

**Discussion by cyiplus** [vuetifyjs/vuetify#15161] [Q&A]
I tried to open the UI Components, API ..., but it can not show to me. would you to check more detail for the web-site.
The web-site more detail as below image:

<img width="541" alt="image" src="https://user-images.githubusercontent.com/82897494/170736314-72b2b2c8-47d4-4c17-825b-5f900062f0aa.png">

**Comment by eachcan** [comment]
I got this for many times.
